cpp/include/cudf/io/orc.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
Hmm. `2019-2020` was consistent with how we're doing it elsewhere.

Reply:
@cwharris This PR depends on #6142 which depends on #6138 which is one of multiple PRs focused on refactoring arguments. The changes you're requesting here are small though - should I make them here? or should they be discussed on PRs related to the refactor where similar changes could maybe be made for other formats @rgsl888prabhu has been working on?